os_net_config/impl_ifcfg.py

```diff
@@ -750,6 +750,11 @@ def _add_common(self, base_opt):
             data += "MTU=%i\n" % base_opt.mtu
         if base_opt.use_dhcpv6 or base_opt.v6_addresses():
             data += "IPV6INIT=yes\n"
+            keep_addr_sysctls = ["net.ipv6.conf.default.keep_addr_on_down=1",
+                                 "net.ipv6.conf.all.keep_addr_on_down=1"]
+            common.write_sysctld_conf("90-v6_keep_addr.conf",
+                                      keep_addr_sysctls)
+            utils.set_keep_addr_sysctl(base_opt.name, self.noop)
```

Comment:
Note that when booting setting net.ipv6.conf.default and net.ipv6.conf.all will ensure that all interfaces get keep_addr_on_down set as well. However while running we must set the value on the interface directly.

Reply:
The v6 routes are also retained when the addresses are retained, so routes come up faster with this in place. Does thst answer your question?